src/Uno.UI.TestComparer/Program.cs

```diff
@@ -160,48 +160,53 @@ IEnumerable<string> GetAllPlatforms()
 
 		private static async Task TryPublishPRComments(List<CompareResult> results, string githubPAT, string sourceRepository, string githubPRid, int currentBuild)
 		{
-			if (!string.IsNullOrEmpty(githubPAT))
+			if (!int.TryParse(githubPRid, out _))
 			{
-				var hasErrors = results.Any(r => r.TotalTests - r.UnchangedTests != 0);
+				Console.WriteLine($"No valid GitHub PR Id found, no PR comment will be posted.");
+				return;
+			}
 
-				if (hasErrors)
-				{
-					var comment = new StringBuilder();
-					comment.AppendLine($"The build {currentBuild} found UI Test snapshots differences.\r\n");
+			if (string.IsNullOrEmpty(githubPAT.Trim()))
+			{
+				Console.WriteLine($"No GitHub PAT, no PR comment will be posted.");
+				return;
+			}
 
-					foreach (var result in results)
-					{
-						comment.AppendLine($"* `{result.Platform}`: **{result.TotalTests - result.UnchangedTests}** changed over {result.TotalTests}\r\n");
+			var hasErrors = results.Any(r => r.TotalTests - r.UnchangedTests != 0);
 
-						var changedTests = result.Tests.Where(t => t.HasChanged).Take(20).ToList();
+			if (hasErrors)
+			{
+				var comment = new StringBuilder();
+				comment.AppendLine($"The build {currentBuild} found UI Test snapshots differences.\r\n");
```

Comment:
Is it possible to add a link to the build directly there?

Reply:
Yes, in a later update :)